jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -51,6 +53,11 @@ def __init__(self):
             '~shutdown_command', '/sbin/shutdown -h now')
         self.reboot_command = rospy.get_param(
             '~reboot_command', '/sbin/shutdown -r now')
+        self.volume = rospy.get_param('~volume', 1.0)
+
+    def speak(self, client, speech_text, lang='jp'):
+        client.say(speech_text, voice=lang, volume=self.volume, replace=False)
```

Comment:
What happens if an action is not executed correctly?
You can also add a timeout to avoid errors if the action is not executed.

Reply:
With timeout, we can use ActionClient if client does not exist. 
However, currently [`wait_for_server()`](https://github.com/ros-drivers/audio_common/blob/27c81ad628430472ddad4e5d063a766c16bf4bfb/sound_play/src/sound_play/libsoundplay.py#L347) is fixed to infinite.

I'm thinking of adding timeout arg to `say()`.
([battery_warning.py](https://github.com/jsk-ros-pkg/jsk_robot/blob/34a129462403878c9b4c9c86c1aa9b38d506c143/jsk_fetch_robot/jsk_fetch_startup/scripts/battery_warning.py#L34) also uses `wait_for_server()`, but it seems to be meaningless...)